pkgs/by-name/xo/xorriso/package.nix

```
@@ -12,11 +12,11 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "xorriso";
-  version = "1.5.7";
+  version = "1.5.7-unstable-2023-11-21";
 
   src = fetchurl {
     url = "https://www.gnu.org/software/xorriso/xorriso-${finalAttrs.version}.tar.gz";
```

This will fail.
Use a web archive link, like
```suggestion
    url = "https://web.archive.org/web/20231206123448/https://www.gnu.org/software/xorriso/xorriso-1.5.7.tar.gz";
```

(And, since I generated this link now, use `-unstable-2023-12-06` as suffix.